servicex/servicex_client.py

```diff
@@ -407,18 +407,10 @@ def cancel_transform(self, transform_id) -> None:
     def get_code_generators(self, backend=None):
         r"""
         Retrieve the code generators deployed with the serviceX instance
+        The backend argument is deprecated and has no effect
```

Comment:
Do we think anyone outside of the cli has ever used this function? Seems not unreasonable to just delete this argument now

Reply:
Yeah, that's cleaner. No need to be too cautious